### DISCUSSION TITLE
Anyone know a fix for game crashing by toggling the render ai pathfinding in Minihud?

### DISCUSSION THREAD

**Discussion by ChickenNdWaffles** [maruohon/minihud#311] [General]
Whenever I try to enable rendering of the AI Pathfinding in MiniHud the game crashes, been like this for a few updates now. I used to use it all the time when moving villagers, now I can't because it crashes the game. Not sure why.

**Comment by maruohon** [comment]
The vanilla code related to the path finding debug renderer has changed a few times between MC versions and I always forget to check or update it...

**Reply by ChickenNdWaffles** [reply]
Hey did you remove the toggle render debug pathfinding in the most recent Minihud update? :( It doesn't seem to be there anymore. Did they add it as a Vanilla feature and I missed it?? I use it all the time and now it's suddenly gone.

**Reply by maruohon** [reply]
Which version are you talking about? I haven't even tried the 1.20.5+ versions so I have no idea about those. It should be in all previous versions at least.

**Comment by ChickenNdWaffles** [comment]
I'm playing on 1.21. You know what? Now that I think about it, I'm using an
unofficial port so any issues aren't on you anyways. 😂 Oops. Lol. Here's
the one I'm using until the official update for 1.21.
https://github.com/sakura-ryoko?tab=repositories

**Comment by Samalando** [comment]
that one is technically the offical port as masa isn't updating to 1.20.5+ and that one is the one masa supports

**Comment by ChickenNdWaffles** [comment]
Well, then the toggle to render debug pathfinding isn't there. Do you know
of another way to turn it on? It's a vanilla feature but it's not normally
accessible by players. Presumably it was used by the devs. People have made
mods that unlock those additional debugs in the past, but I don't know of
any that are up to date. Minihud normally has an option to turn it on but
this new version doesn't.

**Reply by sakura-ryoko** [reply]
I had to disable the feature because it crashes the game, and there is no easy fix for the problem it was causing.

The Vanilla Debug Packets that the code uses are not registered Payloads, so Vanilla Minecraft itself is causing the problem, due to the Payload for the Path Finding Debug packets not being registered.
